src/app/Library/CrudPanel/Traits/Read.php

```diff
@@ -225,6 +232,36 @@ public function addCustomPageLengthToPageLengthMenu()
      */
     public function setPageLengthMenu($menu)
     {
+        // validates the correct building of the menu array
+        if (isset($menu[0]) && ! is_null($menu[0])) {
```

Comment:
```suggestion
        if (isset($menu[0])) {
```

Is this validation needed? The tests run without it.